admin/src/locales/de.json

```diff
@@ -100,6 +104,14 @@
   "form": {
     "cancel": "Abbrechen"
   },
+  "geo-coordinates": {
+    "both-or-none": "Bitte beide oder keine eingeben!",
+    "format": "{latitude}, {longitude}",
+    "label": "Geo-Koordinaten",
+    "latitude-longitude-smart": {
+      "describe": "Teilt Koordinaten im Format 'Breitengrad, Längengrad' automatisch auf. Fügen sie hier einfach z.B. ihre Koordinaten von Google Maps, zum Beispiel: 49.28187664243721, 9.740672183943639, ein."
```

Comment:
Sollte hier nicht besser eine Kurzbeschreibung stehen, dass der Community-Admin die Location-Koordinaten aus dem GMS-System übernehmen soll, die bei der dortigen Anlage der Community erfasst werden?
Denn dies wäre eher der Text, der in den User-Settings-Dialog des Location-Capturings für den User gedacht wäre, wenn er nicht über die Map, sondern die Koordinaten direkt erfassen möchte.